components/compositing/windowing.rs

```diff
@@ -59,7 +59,7 @@ pub enum EmbedderEvent {
     LoadUrl(TopLevelBrowsingContextId, ServoUrl),
     /// Sent when a mouse hit test is to be performed.
     MouseWindowEventClass(MouseWindowEvent),
-    /// Sent when a mouse move.
+    /// Sent when a mouse move. This will send back a `EmbedderMsg::SetCursor` message.
     MouseWindowMoveEventClass(DevicePoint),
```

Comment:
I hope the updated doc comments can clarify the goal of this issue. I think it's reasonable that only the embedder can request the cursor message.

Reply:
I think you are right. I'll close this one and set the cursor_pos instead.